types/devfile.d.ts

```diff
@@ -0,0 +1,15 @@
+export declare class Devfile {
```

Comment:
Git info seems to be missing for sample metadata 🤔 

This is golang's spec for the Git https://github.com/devfile/registry-support/blob/master/index/generator/schema/schema.go#L88

Reply:
fixed

Reply:
```
git?: {
      remotes: {
        origin: string
      }
    }
```

`origin` is not a property field here. It is a custom field and it should probably be `remotes: {}` where key and value are string entries. This can be fixed later